KenanLab1.java

```diff
@@ -1,15 +1,21 @@
 import java.awt.event.*;
+import java.util.*;
 
 public class KenanLab1 implements KeyListener
 {

```

Comment:
You're gonna want to bring those `{` up to definition line. They will get in the way, especially when I grade. Its a style thing. But trust me, when we implement tons of interfaces and you need extra lines to do so, that trailing curly brace on a new line will be cluttered and make things harder to read.